src/modules/customers/components/list/CustomersList.js

```diff
@@ -30,26 +31,37 @@ const propTypes = {
   bulk: PropTypes.array.isRequired,
   emptyBulk: PropTypes.func.isRequired,
   toggleBulk: PropTypes.func.isRequired,
+  toggleAll: PropTypes.func.isRequired,
   addCustomer: PropTypes.func.isRequired,
   history: PropTypes.object
 };
 
 class CustomersList extends React.Component {
+  onChange() {
+    const { toggleAll, customers } = this.props;
+    toggleAll(customers, 'customers');
+  }
+
   renderContent() {
     const { customers, columnsConfig, toggleBulk, history } = this.props;
 
     return (
       <Table whiteSpace="nowrap" hover bordered>
         <thead>
           <tr>
-            <th />
+            <th>
+              <FormControl
+                componentClass="checkbox"
+                onChange={() => this.onChange()}
```

Comment:
Eniig constructor dotor bind hiigeed onChange={this.onChange} geed duudchwal bolmoor yum. chi  onChange={this.onChange()} ingej bicheed render hiihdee ajilaad bga bh. neg shalgaad uzeerei